<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>512</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:rgb(239, 244, 209);}</string>
   </property>
   <widget class="QWidget" name="Container" native="true">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>91</y>
      <width>288</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#Container{
background-color:#1D1E4E;
border:10px solid #1D1E4E;
border-radius: 50px;
}</string>
    </property>
    <widget class="QLabel" name="Login_text">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>16</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Open Sans;
font-style: normal;
font-weight: 800;
font-size: 41px;
line-height: 56px;
/* identical to box height */


color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Login">
     <property name="geometry">
      <rect>
       <x>83</x>
       <y>310</y>
       <width>122</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #00FF85;
border-radius: 20px;
font-family: Open Sans;
font-style: normal;
font-weight: bold;
font-size: 14px;
line-height: 19px;
color: #1D1E4E;
</string>
     </property>
     <property name="text">
      <string>Lets do this</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_username">
     <property name="geometry">
      <rect>
       <x>48</x>
       <y>110</y>
       <width>191</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#input_username{
	font: 12pt &quot;Open Sans&quot;;
background: #FFFFFF;
border-radius: 20px;
}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_password">
     <property name="geometry">
      <rect>
       <x>48</x>
       <y>190</y>
       <width>191</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#input_password{
font: 12pt &quot;Open Sans&quot;;
background: #FFFFFF;
border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="error">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Open Sans&quot;;
color:red;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>233</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFFFFF;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>233</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFFFFF;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>85</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Open Sans;
font-style: italic;
font-weight: normal;
font-size: 15px;
line-height: 39px;

color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>165</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Open Sans;
font-style: italic;
font-weight: normal;
font-size: 15px;
line-height: 39px;

color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Open Sans&quot;;
color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>not registered yet ?, then </string>
     </property>
    </widget>
    <widget class="QPushButton" name="create">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>268</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Open Sans&quot;;
color: #FFCE31;
</string>
     </property>
     <property name="text">
      <string>create one</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label</zorder>
    <zorder>Login_text</zorder>
    <zorder>Login</zorder>
    <zorder>input_username</zorder>
    <zorder>input_password</zorder>
    <zorder>error</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>create</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
